src/services/hasura/hasura.service.ts

```
@@ -426,6 +433,31 @@ export class HasuraService {
       throw new HttpException('Bad request', HttpStatus.BAD_REQUEST);
     }
   }
+  async deleteJobsByBpps(bpps: string[]) {
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)